### DISCUSSION TITLE
How to use this?

### DISCUSSION THREAD

**Discussion by KaSaNaa** [14hz/reddit-karma-farm-bot#2] [Q&A] [answered]
I'd be very happy if you **provide a guide on how to use this script.**

**Comment by 14hz** [comment, marked as answer]
Hello, watch this video: https://youtu.be/5xXuHgZuKLg
create a reddit bot, run it on  replit.com, on replit click on the  + button then import from github and copy paste my project link.

**Reply by KaSaNaa** [reply]
thanks
